### PR TITLE
288 dev committee transfer improving

### PR DESCRIPTION
**Pull request by Katkevich**

### CODE REVIEW

File: libraries/chain/include/scorum/chain/database/database.hpp [hunk 1]
```diff
@@ -189,6 +189,12 @@ class database : public chainbase::database,
     void notify_pre_apply_operation(operation_notification& note);
     void notify_post_apply_operation(const operation_notification& note);
 
+    /**
+     * Note: corresponding signals will be fired iff block is being applying
```
Comment (Andrei-Masilevich) [inline libraries/chain/include/scorum/chain/database/database.hpp]: Is 'iff' typo?

File: libraries/chain/database/database.cpp [hunk 2]
```diff
@@ -1365,6 +1403,16 @@ void database::_apply_block(const signed_block& next_block)
                   "Block produced by witness that is not running current hardfork",
                   ("witness", witness)("next_block.witness", next_block.witness)("hardfork_state", hardfork_state));
 
+        fc::scoped_connection conn_pre(_pre_apply_proposal_operation.connect(
```
Comment (abolinbsu) [inline libraries/chain/database/database.cpp]: I would like to see this logic in plugins.

Comment (Andrei-Masilevich) [inline libraries/chain/database/database.cpp]: Remove unused signal

**Comment by Katkevich** [comment]
Now there is no any extra signals in database.hpp. All notifications for proposal operations go through proposal_virtual_operation.

File: libraries/protocol/include/scorum/protocol/scorum_virtual_operations.hpp [hunk 3]
```diff
@@ -69,12 +69,34 @@ struct comment_reward_operation : public virtual_operation
...
-    fill_vesting_withdraw_operation()
+    acc_total_vesting_withdraw_operation() = default;
+    acc_total_vesting_withdraw_operation(const std::string& f, const asset& w)
+        : from_account(f)
+        , withdrawn(w)
+    {
+    }
+
+    account_name_type from_account;
+    asset withdrawn = asset(0, SP_SYMBOL);
```
Comment (abolinbsu) [inline libraries/protocol/include/scorum/protocol/scorum_virtual_operations.hpp]: Is withdrawn actually used? Is it total amount of withdraw?

File: libraries/protocol/include/scorum/protocol/operations.hpp [hunk 4]
```diff
@@ -60,14 +60,20 @@ using operation = fc::static_variant<vote_operation,
...
-                                     fill_vesting_withdraw_operation,
                                      hardfork_operation,
                                      producer_reward_operation,
                                      active_sp_holders_reward_operation,
                                      return_scorumpower_delegation_operation,
                                      shutdown_witness_operation,
                                      witness_miss_block_operation,
-                                     expired_contract_refund_operation>;
+                                     expired_contract_refund_operation,
+                                     acc_total_vesting_withdraw_operation,
```
Comment (abolinbsu) [inline libraries/protocol/include/scorum/protocol/operations.hpp]: acc_total_vesting_withdraw_operation and devpool_total_vesting_withdraw_operation actually means acc_vesting_withdraw_finish_operation and devpool_vesting_withdraw_finish_operation. Could you rename it?

**Pull request merged**